src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ CHANGELOG
  * Not passing the project directory to the constructor of the `AssetsInstallCommand` is deprecated. This argument will
    be mandatory in 5.0.
  * Deprecated the "Psr\SimpleCache\CacheInterface" / "cache.app.simple" service, use "Symfony\Contracts\Cache\CacheInterface" / "cache.app" instead
+ * Added an `help_html` form option to display the `help` text as HTML
```

Comment:
this change doesn't look related

Reply:
Bad conflict resolution while rebasing, my bad